core/app/models/spree/stock/package.rb

```diff
@@ -11,7 +11,8 @@ def initialize(stock_location, contents=[])
       end
 
       def add(inventory_unit, state = :on_hand)
-        contents << ContentItem.new(inventory_unit, state) unless find_item(inventory_unit)
+        # Remove find_item check as already taken care by prioritizer
```

Comment:
Removed find_item(inventory_unit) as this scenario is already handled by Prioritizer.